### PR TITLE
Main with 2.3.4 to develop

### PR DESCRIPTION
**Pull request by andreievg**
This PR branch into develop: https://github.com/msupply-foundation/open-msupply/pull/5709

### CODE REVIEW

File: client/packages/common/src/intl/locales/tet/replenishment.json [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+  "messages.confirm-delete-requisition-lines_one": "Ne'e sei hasai permanente lina 1 husi pedidu ida ne'e",
+  "messages.confirm-delete-requisition-lines_other": "Ne'e sei hasai permanente {{count}} linas husi pedidu ida ne'e",
+  "messages.confirm-delete-requisitions_one": "Ne'e sei hasai permanente pedidu internal 1",
+  "messages.confirm-delete-requisitions_other": "Ne'e sei hasai permanente {{count}} pedidu internal",
+  "messages.confirm-delete-shipment-lines_other": "Ne'e sei hasai permanente {{count}} linas remessa",
+  "messages.deleted-requisitions_one": "Hamos ona {{count}} pedidu",
+  "messages.deleted-requisitions_other": "Hamos ona {{count}} pedidu sira",
+  "messages.confirm-status-as": "Konfirma estatus hanesan $t({{status}})?",
+  "messages.saved": "Salva ona",
+  "label.rnr-adjustments": "Ajustamentu",
```
Comment (jmbrunskill) [inline client/packages/common/src/intl/locales/tet/replenishment.json:89]: I think we need to move these to common now for `develop`?

**Review by jmbrunskill — CHANGES_REQUESTED** [review]
I think replenishment.json no longer exists in develop.
This might effectively loose the tetum translations?

File: client/packages/common/src/intl/locales/tet/replenishment.json (hunk 1 above)
Comment (roxy-dao) [reply client/packages/common/src/intl/locales/tet/replenishment.json:89]: Oh right nice catch James XD

File: client/packages/common/src/intl/locales/tet/common.json [hunk 2]
```diff
@@ -569,5 +568,18 @@
...
+  "label.rnr-initial-balance": "Balansu Inisiu",
+  "label.rnr-maximum-quantity": "Kuantidade maximu",
+  "label.rnr-minimum-quantity": "Kuantidade minimu",
+  "label.rnr-received": "Kuantidade Simu",
+  "label.rnr-stock-out-duration": "Durasaun stock out",
+  "label.adjusted": "Ajustamentu kuantitade konsumu / distribuída",
+  "label.low-stock": "Stock menus",
+  "label.approved-quantity": "Kuantidade aprova",
+  "label.losses": "Lakon",
+  "label.amc": " Konsumu / Distribuída mediu mensal"
```
Comment (jmbrunskill) [inline client/packages/common/src/intl/locales/tet/common.json:584]: This possibly should be changed to an R&R specific one?

Comment (jmbrunskill) [reply client/packages/common/src/intl/locales/tet/common.json:584]: But I think it's ok to merge like this...

Comment (jmbrunskill) [reply client/packages/common/src/intl/locales/tet/common.json:584]: There's plans for a more thorough Tetum translation to be done in the future

**Review by jmbrunskill — APPROVED** [review]
Just pushed a fix for the `common.json` change.

**Pull request merged**